cleanup-passes/travis.ts

```diff
@@ -88,12 +88,12 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
   const ta = travis.addons;
 
   // use latest firefox, unless specified
-  if (!ta.firefox) {
-    ta.firefox = 'latest';
-  }
+  ta.firefox = 'latest';
```

Comment:
This looks like it always overwrites, which isn't what the comment says. Update the comment?